### PR TITLE
Update kafka chart to 22.1.3 for kafka 3.4.0

### PR DESCRIPTION
**Pull request by jinlintt**
Kafka chart `17.1.0` is already more than one year old. Updating it to the latest so we can also get the latest kafka.
Still default to using zookeeper.

https://github.com/bitnami/charts/pull/16778

We have deployed this version of the chart to an existing cluster in GKE and kafka worked fine afterwards.

The corresponding docker change is in the PR below.
https://github.com/datahub-project/datahub/pull/8161

### CODE REVIEW

**Comment by jinlintt** [comment]
@david-leifker could you please take a look?

File: charts/prerequisites/values.yaml [hunk 1]
```diff
@@ -88,3 +88,7 @@ cp-helm-charts:
 # Bitnami version of Kafka that deploys open source Kafka https://artifacthub.io/packages/helm/bitnami/kafka
 kafka:
   enabled: true
+  kraft:
```
Comment (Santhin) [inline charts/prerequisites/values.yaml:91]: I think there is typo?

Comment (Santhin) [reply charts/prerequisites/values.yaml:91]: My bad

**Comment by david-leifker** [comment]
Thanks!

**Pull request merged**